tests/product/test_control.py

```diff
@@ -65,37 +65,10 @@ def test_server_start_one_host_started(self):
         self.assert_start_with_one_host_started(
```

Comment:
Can you change the test method name in this test (line 63) to match the test method name on line 68? Either both should use "one host" or "node".